forc-plugins/forc-crypto/src/keygen/new_key.rs

```diff
@@ -13,9 +13,17 @@ use libp2p_identity::{secp256k1, Keypair, PeerId};
 use serde_json::json;
 use std::ops::Deref;
 
+const ABOUT: &str = "Creates a new key for use with fuel-core";
+
+forc::cli_examples! {
+    [ Creates a new key for block production by default => crypto "new-key" ]
+    [ Creates a new key for peering => crypto "new-key" "-k" "peering" ]
+    [ Creates a new key for block production => crypto "new-key" "-k" "peering" ]
```

Comment:
```suggestion
    [ Creates a new key for block production and peering => crypto "new-key" "-k" "peering" ]
```

Same command for two behaviours merged into 1